app/models/Search.java

```diff
@@ -323,8 +329,13 @@ private SearchHits getTotalHits(final SearchResponse response) {
 		return hits;
 	}
 
-	private SearchResponse startInitialResponse() {
-		final QueryBuilder queryBuilder = createQuery();
+	private SearchResponse startInitialResponse(final String scroll) {
+		QueryBuilder queryBuilder = createQuery();
+		if (scroll.matches("\\d{8}")) {
+			final QueryBuilder changedSinceQuery =
+					new LobidResources.ChangedSinceQuery().build(scroll);
+			queryBuilder = boolQuery().must(queryBuilder).must(changedSinceQuery);
+		}
```

Comment:
I think this should go into the `createQuery` method, where the other special parameters are handled and added to the resulting full query.